### ISSUE TITLE
1.5.0 is crashing on 1.19.2 startup

### ISSUE THREAD

**Issue by fuzzyweapon** [Deftu/DeftuLib#3]
https://www.curseforge.com/minecraft/mc-mods/deftulib/files/4396069

gave me the Fabric Loader error on launch:

```
A mod crashed on startup!
net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'deftulib'!
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:164)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.RuntimeException: Could not execute entrypoint stage 'preLaunch' due to errors, provided by 'deftulib'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:162)
	... 2 more
Caused by: java.lang.NoClassDefFoundError: okio/Buffer
	at okhttp3.ResponseBody$Companion.create(ResponseBody.kt:248)
	at okhttp3.ResponseBody$Companion.create$default(ResponseBody.kt:247)
	at okhttp3.internal.Util.<clinit>(Util.kt:65)
	at okhttp3.internal.concurrent.TaskRunner.<clinit>(TaskRunner.kt:309)
	at okhttp3.ConnectionPool.<init>(ConnectionPool.kt:41)
	at okhttp3.ConnectionPool.<init>(ConnectionPool.kt:47)
	at okhttp3.OkHttpClient$Builder.<init>(OkHttpClient.kt:471)
	at xyz.deftu.lib.telemetry.TelemetryTracker.recordPreLaunch(TelemetryTracker.kt:48)
	at xyz.deftu.lib.prelaunch.DeftuLibPreLaunch.onPreLaunch(DeftuLibPreLaunch.kt:62)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 4 more
Caused by: java.lang.ClassNotFoundException: okio.Buffer
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 14 more
```

Full log below...
https://hastebin.com/share/ucatatemix.bash

**Comment by Deftu** [comment]
Oops, I forgot to bundle the okio library which OkHttp requires. I'll release a patch for that in a few minutes.

**Comment by Deftu** [comment]
This issue is now fixed as of 46dc8a5418f2f23183e456bfc5760c52bc25c483.
